src/mscorlib/shared/System/IO/DriveNotFoundException.cs

```diff
@@ -7,6 +7,8 @@
 namespace System.IO
 {
     //Thrown when trying to access a drive that is not available.
+    [Serializable]
+    [System.Runtime.CompilerServices.TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
     internal class DriveNotFoundException : IOException
```

Comment:
The actual public DriveNotFoundException lives in corefx...

Reply:
This we actually do throw in one place. So I think strictly, we would want it to be serializable even if not deserializable? The serialized blob would look like the public one.

Reply:
Same as AppDomainUnloadedException